token/program-2022/src/extension/confidential_transfer/instruction.rs

```diff
@@ -225,16 +225,71 @@ pub enum ConfidentialTransferInstruction {
     ///   None
     ///
     DisableBalanceCredits,
+
+    /// Transfer all withheld confidential tokens in the mint to an account. Signed by the mint's
+    /// withdraw withheld tokens authority.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   0. `[writable]` The token mint. Must include the `TransferFeeConfig` extension.
+    ///   1. `[writable]` The fee receiver account. Must include the `TransferFeeAmount` and
+    ///      `ConfidentialTransferAccount` extensions.
+    ///   2. `[]` Instructions sysvar
+    ///   3. `[signer]` The mint's `withdraw_withheld_authority`.
+    /// or:
+    ///   3. `[]` The mint's `withdraw_withheld_authority`'s multisignature owner/delegate.
+    ///   4. ..3+M `[signer]` M signer accounts.
+    ///
+    /// Data expected by this instruction:
+    ///   WithdrawWithheldTokensFromMintData
+    ///
+    WithdrawWithheldTokensFromMint,
+
+    /// Transfer all withheld tokens to an account. Signed by the mint's withdraw withheld tokens
+    /// authority. This instruction is susceptible to front-running. Use
```

Comment:
" This instruction is susceptible to front-running." - Another sentence providing a bit more detail on when/how front running might occur here would help users

Reply:
Right. Should definitely add details here.